akka-actor/src/main/resources/reference.conf

```diff
@@ -438,6 +441,16 @@ akka {
       mailbox-requirement = ""
     }
 
+    default-blocking-io-dispatcher {
+      type = "Dispatcher"
+      executor = "thread-pool-executor"
+      throughput = 1
+
+      thread-pool-executor {
+        fixed-pool-size = 16
+      }
+    }
```

Comment:
Haha, now I see that was from Patriks comment, sorry missed that, ignore this question.